gp-templates/helper-functions.php

```diff
@@ -166,10 +197,10 @@ function map_glossary_entries_to_translation_originals( $translation, $glossary,
 		$escaped_chunk = esc_translation( $chunk );
 
 		// Create a lower case version to compare with the glossary terms.
-		$lower_chunk = strtolower( $chunk );
+		$lower_chunk = strtolower( preg_quote( $chunk, '/' ) );
 
 		// Search the glossary terms for a matching entry.
-		if ( false !== array_search( $lower_chunk, $glossary_entries_terms_array, true ) ) {
+		if ( in_array( $lower_chunk, $glossary_entries_terms_array ) ) {
```

Comment:
Why is this no longer a strict check?

Reply:
Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1359/commits/0c7959d8030d1207da5de831b840a342b8025d40